sql/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -1948,5 +1949,32 @@ public void testNegationOfNonNumericLiteralsShouldFail() throws Exception {
         analyze("select - 'foo'");
     }
 
+    @Test
+    public void testIsNullPredicateIsAllowed() {
```

Comment:
would probably make sense to create one test which tests that predicates are allowed. eg testPredicatesInSelectList and then do something like "select 1 is null, 1 is not null, not false;"